src/components/map/MapFilter.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Input } from 'antd';
+import { AudioOutlined } from '@ant-design/icons';
+import styled from 'styled-components';
+import axios from 'axios';
+import Map from '../map/Map';
+
+const StyledDiv = styled.div`
+  margin: 2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+
+const MapFilter = () => {
+  useEffect(() => {
+    axios
+      .get('http://localhost:5000/')
+      .then(res => {
+        dispatch({ type: 'GETDATA', payload: res.data });
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
No console logs in production. We need to find a different way to error handle.

Reply:
Okay, will take the console logs out. What would be a different way to error handle?

Reply:
In a production environment you'd be sending this error to your logging system, perhaps generating a notification that goes to the dev team. What you do with the error is going to vary greatly. In this case, you might cause a pop up for the user that informs them that there was an issue with whatever request they made so that they can re-attempt it or contact someone about their issue. This is especially the best option if the error is the result of some input from the user that might not have been what was required or expected.

Reply:
Got it. Changes made and requested re-review.